css/_font.scss

```diff
@@ -25,6 +25,15 @@
     -moz-osx-font-smoothing: grayscale;
 }
 
+.icon-public:before {
+  content: "\e80b";
+}
+.icon-event_note:before {
+    content: "\e616";
+}
+.icon-timer:before {
+    content: "\e425";
+}
```

Comment:
:notesureif: What does the work on the mobile recent-list feature have to do with Web?

Reply:
I just added the 3 new icons according to the doc, and it involves the web part as well - which I think makes sense, cause I guess we don't want to have a set of icons being on mobile and another subset on web (?)